applications/crossbar/priv/couchdb/schemas/auth.callback.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "_id": "auth.callback",
+    "additionalProperties": true,
+    "description": "callback for authentication providers",
+    "properties": {
+        "clientId": {
+            "description": "client id, usually appplication id for oauth providers",
+            "type": "string"
+        },
+        "code": {
+            "description": "access code emitted by provider",
+            "type": "string"
+        },
+        "provider": {
+            "description": "provider",
+            "type": "string"
+        },
+        "redirecUri": {
```

Comment:
well, i followed the javascript library that sends these fields.  not sure if we should change that library